packages/langium/src/test/langium-test.ts

```diff
@@ -16,8 +16,9 @@ import { LangiumDocument } from '../workspace/documents';
 import { findNodeForProperty } from '../utils/grammar-util';
 import { SemanticTokensDecoder } from '../lsp/semantic-token-provider';
 import { TextDocument } from 'vscode-languageserver-textdocument';
+import { LangiumGrammarServices } from '../grammar/langium-grammar-module';
 
-export function parseHelper<T extends AstNode = AstNode>(services: LangiumServices): (input: string) => Promise<LangiumDocument<T>> {
+export function parseHelper<T extends AstNode = AstNode>(services: LangiumGrammarServices): (input: string) => Promise<LangiumDocument<T>> {
```

Comment:
These testing utilities are not only meant for our own grammar language, but are published to by used for any Langium-based language. Please revert the changes in this file, even if it means we have to type-cast in our grammar language tests.